var/spack/repos/builtin/packages/pgplot/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pgplot(MakefilePackage):
+    """PGPLOT Graphics Subroutine Library
+
+    The PGPLOT Graphics Subroutine Library is a Fortran- or
+    C-callable, device-independent graphics package for making
+    simple scientific graphs. It is intended for making
+    graphical images of publication quality with minimum effort
+    on the part of the user. For most applications, the program
+    can be device-independent, and the output can be directed to
+    the appropriate device at run time."""
+
+    homepage = "https://sites.astro.caltech.edu/~tjp/pgplot/"
+    url      = "ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz"
+
+    maintainers = ['eschnett']
+
+    version('5.2.2',
+            url="ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz",
+            sha256='a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4')
+
+    # Create a file `conf`. This is how pgplot is configured, defining
+    # specifying compilers, paths, etc. We just point to the respective Spack
+    # wrappers.
+    patch('conf')
+
+    parallel = False
+
+    def build(self, spec, prefix):
+        makemake = which('./makemake')
+        makemake(self.build_directory, 'linux', 'g77_gcc')
+        make()
+        make('clean')
+        make('cpg')
+
+    def install(self, spec, prefix):
+        mkdirp(prefix.bin)
```

Comment:
I didn't inspect the package, but can we use `install_tree` instead of installing single files? Or are there issues with that?

Reply:
Yes we can! Done.

Reply:
I think this was fixed only for `lorene` and not for `pgplot`

Reply:
pgplot puts all created files into the same directory. install_tree doesn't have a way to list files that should be installed. I think using install_tree would be awkward here.

Reply:
Ugh, thanks for pointing out I missed that detail.